.gitignore

```diff
@@ -309,3 +309,7 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+.DS_Store
```

Comment:
instead of 4 entries, just

```
**/.DS_Store
```

should work for every folder? and add a comment here what/why

Reply:
removed .gitignore from this PR as it was added by accident.